run-rsinstallpackagebuilder/src/index.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable max-len */
+/* eslint-disable dot-notation */
+/* eslint-disable no-lonely-if */
+const tc = require('@actions/tool-cache');
+const io = require('@actions/io');
+const fs = require('fs');
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+const path = require('path');
+const uuidV4 = require('uuid/v4');
+const https = require('https');
+const { getBinaryName } = require('./pkgbuilder');
+const { checkEnv } = require('../../utils');
+
+const packageBuilderCommand = async (
+  builder, packageName, workingDir, outputDir, sourcePaths, sourceFilePaths) => exec.exec(builder,
+  ['--pn', packageName, '--wd', workingDir, '--od', outputDir, '--sp', sourcePaths, '--sfp', sourceFilePaths]);
+
+function downloadTool(url) {
```

Comment:
Is this downloadTool re-implementation needed? Why?
Also, what changes are done here that belongs in the utils file? We should not merge as-is.